### PR TITLE
Reduce runtime of `test-index-from-database` from 26s to 6s

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

**Comment by yjpa7145** [comment]
> I would like to have this tested with more than 1

Looking at the `indexFromDatabase` function, I don't see that it would handle 10 any differently than 1. What case am I not thinking of?

**Comment by yjpa7145** [comment]
@laurenfrederick nudge

File: packages/api/tests/models/test-granules-model.js [hunk 1]
```diff
@@ -449,46 +447,39 @@ test('scan() will translate old-style granule files into the new schema', async
...
-  // get all completed v1 granules
-  const queueV1Completed = granule.getGranulesForCollection(collectionIds[0], 'completed');
-  const sortedV1CompletedGranIds = granulesV1.map((gran) => gran.granuleId).sort();
-  for (let i = 0; i < 5; i += 1) {
-    const gran = await queueV1Completed.peek(); // eslint-disable-line no-await-in-loop
-    t.is(gran.granuleId, sortedV1CompletedGranIds[i]);
-    t.is(gran.collectionId, collectionIds[0]);
-    queueV1Completed.shift();
-  }
+test('getGranulesForCollection() only returns granules belonging to the specified collection', async (t) => {
```
Comment (laurenfrederick) [inline packages/api/tests/models/test-granules-model.js]: This seems to be testing sorting as well. Is this tested somewhere else?

**Comment by yjpa7145** [comment]
> I was thinking it'd be nice to have more than 1 item in the queue to make sure we're iterating through it.

Testing to make sure that the `DynamoDbSearchQueue` handles paging is done here: https://github.com/nasa/cumulus/blob/master/packages/common/tests/aws/DynamoDbSearchQueue.js#L73

File: packages/api/tests/models/test-granules-model.js (hunk 1 above)
Comment (yjpa7145) [reply packages/api/tests/models/test-granules-model.js]: Added test for that here: fef57a490

**Pull request merged**